### DISCUSSION TITLE
Update the website or transfer its control

### DISCUSSION THREAD

**Discussion by kopeboy** [rickyhewitt/nyano.org#19] [General]
Nyano is a good brand cause Nyan cat is a good, timeless meme.
Right now, with the current content at nyano.org, I think you are doing more harm than good to the community of Nano supporters.

If you can't keep the site updated, with working links and a minimal explanation of what Nyano & Nano are, I suggest you transfer the control of the domain to some active member(s) of the nano community.

If you want, I am available.
An alternative would be just to put the site in a state that even if not updated can give timeless & useful info to nano & nyan cat curious people :) Something like a paragraph on the history of the project and a link to nano.org & [nano.org/wallets](https://hub.nano.org/wallets) instead of broken links or link to abandoned repos or 404 domains.

🙏🏻

I hope to hear from you! 🤗

**Comment by rickyhewitt** [comment]
Hi Lorenzo,

The domain itself isn't something I have control of, so there isn't much I can personally do in the way of a domain transfer. I only registered the domain for 1 year in 2021, It was already transferred to someone else internally within NameCheap in 2022 & they just left it pointing to my CloudFlare (& Netlify). At the time of the transfer the links were still working. 

Unfortunately I no longer have any record of whom it was transferred to, and NameCheap doesn't appear to provide any way of finding out. 

I can still make changes (for now). The difficulty with fixing the links is that the upstream projects have either disappeared or become stagnant themselves, and as I have no access to the domain itself I made a decision to just leave the site as-is.

Since the domain expires in 2 months, it may be best to just try to buy it if it becomes available again.. & if you have any changes you wish to integrate in the mean time feel free to submit them as a pull request (or alternatively if it is just text, then feel free to submit as an issue / feature request, or even just a comment below).

_whois:_
<img width="1017" alt="Screenshot 2024-08-12 at 12 24 53" src="https://github.com/user-attachments/assets/fec368d9-fd6a-4568-b170-a05489f2ac88">

_order history 1:_
<img width="774" alt="Screenshot 2024-08-12 at 12 39 17" src="https://github.com/user-attachments/assets/11faf81d-6969-48cc-a6f1-00b4c77a0c8e">

_order history 2:_
<img width="624" alt="Screenshot 2024-08-12 at 12 39 45" src="https://github.com/user-attachments/assets/00dab44c-9615-451e-9b85-d90cfbb94ff6">

**Comment by kopeboy** [comment]
Thanks a lot for all the infos!
Well, I'd say if it doesn't bother you much, we could try at least removing the broken links and adding a paragraph like this:

> Nyano is a meme about the famous Nyan Cat and the glorious cryptocurrency Nano. 
> Web & mobile wallets were created to show 1 nyano = 10^-6 XNO (Nano's canonical denomination) to let people play with the feeless digital currency in small amounts, but are now abandoned. You can find the original codebase and some useful references [here](https://github.com/rickyhewitt/nyano.org). You can ofc still use the Nano payment network, which is stronger than ever, just by adding some zeros after the decimal separator 😊 Refer to [nano.org](https://nano.org/) to find the currently supported wallets as well as exchanges allowing you to swap it from your local traditional currency.

I'll set a reminder to check if the domain becomes available in November 😉

Btw, what are you up to nowadays? Did you know Nano is about to be upgraded to v27, which includes strong spam protection & greatly increases the throughput? 🤓😇

**Reply by rickyhewitt** [reply]
Not sure if I already replied elsewhere, but yes, have gone ahead with that.

I have been unfortunately very busy and so wasn't able to respond as fast as I'd liked. Have recently started my degree and been swamped with existing work. Thanks for asking though! :) 

I'm no longer involved with the nano community, although I wish them the best of luck.

**Comment by kopeboy** [comment]
Thanks for updating. I noticed you missed adding the links though 😇😬

If possible, make the "here" as a link to this repo: `<a href="https://github.com/rickyhewitt/nyano.org">here</a>` 🙏🏻

Nano.org is self-eplanatory, but once you're there... substitute it with `<a href="https://nano.org">nano.org</a>` 😊

**Reply by rickyhewitt** [reply]
Gone ahead and added those. Thanks.
